app/locales/ml.json

```diff
@@ -0,0 +1,113 @@
+{
+  "label": {
+    "about_title": "വിവരങ്ങൾ",
+    "authorities_add_button_label": "വിശ്വസനീയമായ ഉറവിടം ചേർക്കുക",
+    "authorities_add_url": "URL വഴി ഉറവിടം ചേർക്കുക",
+    "authorities_desc": "എക്‌സ്‌പോഷർ ഡാറ്റ നേടുന്നതിന് നിങ്ങളുടെ പ്രദേശത്തെ വിശ്വസനീയ ആരോഗ്യ സംരക്ഷണ അധികാരികളെ തിരഞ്ഞെടുക്കുക. ഒന്നുകിൽ ആഗോള രജിസ്ട്രിയിൽ നിന്ന് ഒരു പേര് തിരഞ്ഞെടുക്കുക, അല്ലെങ്കിൽ സെയ്ഫ് പാത്ത് നടപ്പിലാക്കിയ ഒരു അതോറിറ്റി നൽകിയ വെബ് വിലാസം നൽകുക.",
+    "authorities_input_placeholder": "നിങ്ങളുടെ URL ഇവിടെ ചേർക്കുക",
+    "authorities_no_sources": "ഇതുവരെ ഡാറ്റ ഉറവിടങ്ങളൊന്നുമില്ല",
+    "authorities_removal_alert_cancel": "റദ്ദാക്കുക",
+    "authorities_removal_alert_desc": "ഈ അതോറിറ്റി ഡാറ്റ ഉറവിടം നീക്കംചെയ്യണമെന്ന് നിങ്ങൾക്ക് ഉറപ്പാണോ?",
+    "authorities_removal_alert_proceed": "തുടരുക",
+    "authorities_removal_alert_title": "അതോറിറ്റി(ഉറവിടം) നീക്കംചെയ്യുക",
+    "authorities_title": "വിശ്വസനീയമായ ഉറവിടങ്ങൾ",
+    "choose_provider_subtitle": "എക്‌സ്‌പോഷറുകളെക്കുറിച്ച് അറിയാൻ നിങ്ങൾ ഒരു ഹെൽത്ത് അതോറിറ്റിയിലേക്ക് സബ്‌സ്‌ക്രൈബു ചെയ്യേണ്ടതുണ്ട്.",
+    "choose_provider_title": "ആരോഗ്യ അതോറിറ്റി തിരഞ്ഞെടുക്കുക",
+    "commitment": "പ്രതിബദ്ധത",
+    "commitment_para": " നിങ്ങളുടെ സെയ്ഫ് പാത്ത് ഉപയോഗിക്കുന്ന ആളുകളുമായുള്ള നിങ്ങളുടെ ഇടപെടൽ സുരക്ഷിതമായി രേഖപ്പെടുത്തുകയും പരിശോധിക്കുകയും ചെയ്യുന്നു. നിങ്ങളുടെ സമ്മതമില്ലാതെ നിങ്ങളുടെ ഡാറ്റ ഒരിക്കലും ഫോൺ വിടുകയില്ല.",
+    "default_news_site_name": "സെയ്ഫ് പാത്ത് വാർത്തകൾ",
+    "default_news_site_url": "https://covidsafepaths.org/in-app-news",
+    "event_history_subtitle": "ആരോഗ്യ അധികാരികൾ പങ്കിട്ട വിവരങ്ങളെ അടിസ്ഥാനമാക്കി നിങ്ങളുടെ എക്സ്പോഷർ മനസ്സിലാക്കുക.",
+    "event_history_title": "എക്സ്പോഷർ ചരിത്രം",
+    "export_para_1": "നിങ്ങൾ COVID-19  പോസിറ്റീവ് ആണെന്ന് കണ്ടെത്തിയാൽ, ദയവായി നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രം പ്രാദേശിക അധികാരികളുമായി പങ്കിടുക. ഇതിലൂടെ നിങ്ങൾ നിങ്ങളുടെ ഭാഗം ചെയ്യുന്നു .",
+    "export_para_2": " സമയത്തിന്റെയും സ്ഥലങ്ങളുടെയും ലളിതമായ ലിസ്റ്റായി ലൊക്കേഷൻ പങ്കിടുന്നു, അധിക വിവരങ്ങളൊന്നുമില്ല.",
+    "home_at_risk_header": "You May Be Exposed",
+    "home_at_risk_subsubtext": "നിങ്ങൾ രോഗബാധിതനാണെന്ന് ഇതിനർത്ഥമില്ല",
+    "home_at_risk_subtext": "നിങ്ങളുടെ ജി‌പി‌എസ് ചരിത്രത്തെ അടിസ്ഥാനമാക്കി, നിങ്ങൾ‌ COVID19 രോഗനിർണയം നടത്തിയ ഒരാളുമായി സമ്പർക്കം പുലർത്തുകയോ അല്ലെങ്കിൽ‌ അവരുമായി അടുത്തിടപഴകുകയോ ചെയ്‌തിരിക്കാം.",
+    "home_enable_location": "ലൊക്കേഷൻ ഡാറ്റ അനുവദിക്കുക",
+    "home_mayo_link_heading": "കൂടുതൽ COVID-19 വിവരങ്ങൾ",
+    "home_mayo_link_label": "മയോ ക്ലിനിക്കിൽ നിന്ന്",
+    "home_mayo_link_URL": "https://www.mayoclinic.org/coronavirus-covid-19",
```

Comment:
we remove the URLs and let them fall back to the default. @tstirrat can confirm that.

Reply:
@alpita-masurkar , @tstirrat   : shall I remove all of them? (default_news_site_url, home_mayo_link_URL, and terms_of_use_url) from ml.json ?

Reply:
Yes please.

Remove the entire entry (don't leave as "", this will cause it to translate to an empty string)